Apps/Inc/ReadTritium.h

```diff
@@ -25,6 +25,7 @@ typedef enum{
     T_DESAT_FAULT_ERR = (1<<7), 
     T_MOTOR_OVER_SPEED_ERR = (1<<8), 
     T_INIT_FAIL = (1<<9), //motor controller fails to restart or initialize
+    T_MOTOR_WATCHDOG_TRIP = (1 << 15), // Defined by us, not by tritium

```

Comment:
Could we mention in the comment where these are defined? Like say, "These are defined in the tritium motor doc called xxx"?

Reply:
Yeah I can add a comment